rest-api-cloud-client/src/main/java/com/stnetix/ariaddna/externalcloudapi/implementation/DropBoxHelper.java

```diff
@@ -0,0 +1,234 @@
+package com.stnetix.ariaddna.externalcloudapi.implementation;
+
+import com.dropbox.core.DbxAppInfo;
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.v2.DbxClientV2;
+import com.dropbox.core.v2.files.*;
+import com.dropbox.core.v2.users.SpaceUsage;
+import com.google.gson.JsonObject;
+import com.sun.jndi.toolkit.url.Uri;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.*;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.util.Locale;
+
+/**
+ * Created by LugovoyAV on 06.09.2017.
+ */
+//запрос на аторизацию
+//https://www.dropbox.com/1/oauth2/authorize?response_type=code&client_id=ioiju89iyfxjy4h
+//TODO непонятно пока что возвращать JsonObject
+public class DropBoxHelper {
+    final static String APP_KEY = "ioiju89iyfxjy4h";
```

Comment:
Please don't include private information on public resource!

Reply:
Where we store private information?

Reply:
We have keystore  for certificates and database for other system information.